### PR TITLE
image-sets-view: fixes image-sets-view sort_by updated-at.

### PR DESCRIPTION
**Pull request by ldjebran**
Fixes THEEDGE-1553

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [X] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/routes/imagesets_test.go [hunk 1]
```diff
@@ -905,6 +905,21 @@ var _ = Describe("ImageSets Route Test", func() {
 							Expect(imageSetsViewResponse.Data[i].Name <= imageSetsViewResponse.Data[i+1].Name).To(BeTrue())
 						}
 					}
+					if sort_arg == "-name" {
```
Comment (omaciel) [inline pkg/routes/imagesets_test.go:908]: Perhaps using a switch statement would make the code clearer, but it's only a suggestion of course

**Review by omaciel — DISMISSED** [review]
/lgtm

**Review by tomershinhar — APPROVED** [review]
LGTM

File: pkg/routes/imagesets_test.go (hunk 1 above)
Comment (ldjebran) [reply pkg/routes/imagesets_test.go]: updated :smiley:

**Review by tomershinhar — APPROVED** [review]

**Review by omaciel — APPROVED** [review]
/lgtm

**Review by de1987 — APPROVED** [review]

**Pull request merged**